utils/post_utils.jsx

```diff
@@ -24,6 +24,10 @@ export function isSystemMessage(post) {
     return Boolean(post.type && (post.type.lastIndexOf(Constants.SYSTEM_MESSAGE_PREFIX) === 0));
 }
 
+export function isMeMessage(post) {
```

Comment:
We could just call the redux version of this function instead of duplicating it here

Reply:
Agreed, was mostly waiting for the PR in the redux repo to go through

Reply:
I have removed the duplicated implementation of isMeMessage